doc/dash/dash-sonic-hld.md

```diff
@@ -672,14 +692,10 @@ DASH_PA_VALIDATION_TABLE:{{vni}}
 ```
 key                      = DASH_PA_VALIDATION_TABLE:vni; ENI and VNI as key;
 ; field                  = value
-addresses                = list of addresses used for validating underlay source ip of incoming packets. 
+addresses                = list of prefixes used for validating underlay source ip of incoming packets. 
```

Comment:
Should we rename the field to `prefixes` as well?

Reply:
done